builds/misc/packages.yaml

```diff
@@ -99,3 +99,54 @@ jobs:
           PathtoPublish: '$(build.artifactstagingdirectory)'
           ArtifactName: 'iotedged-$(os)-$(arch)'
         condition: succeededOrFailed()
+
+################################################################################
+  - job: mariner_linux
+################################################################################
+    displayName: Mariner_Linux
+    pool:
+      vmImage: 'ubuntu-18.04'
+    variables:
+      NugetSecurityAnalysisWarningLevel: warn
+    strategy:
+      matrix:
+        CBL-Mariner1.0-amd64:
+          arch: amd64
+          os: mariner
+          release: tags/1.0-stable
```

Comment:
Do you guys usually tag something against your branch when you ingress the new version of IoTEdge?